common/src/main/java/io/druid/metadata/MetadataStorageConnectorConfig.java

```diff
@@ -38,8 +38,8 @@
   @JsonProperty
   private String user = null;
 
-  @JsonProperty
-  private String password = null;
+  @JsonProperty("password")
```

Comment:
Yes, there are many other things we need to add to that list, we can do that in a separate PR.